src/components/ReportActionItem/MoneyRequestAction.tsx

```diff
@@ -92,7 +92,7 @@ function MoneyRequestAction({
         }
 
         // If the childReportID is not present, we need to create a new thread
-        const childReportID = action?.childReportID ?? '0';
+        const childReportID = action?.childReportID ?? 0;
```

Comment:
```suggestion
        const childReportID = action?.childReportID;
```

It is wrong for a "reportID" to be a number, the default used before is a bad default. The new default`'0'` is also not good because it makes `!childReportID` evaluate to `false`. I think if this is missing, it is better to stay as undefined or null, but not 0 or '0'